Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,18 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+A > O(n)
+
+B> O(n4)
```

Comment:
```for l in range(k + 1, 10 + k):```
k doesn't increment inside of this loop, so the loop will run a specific number of times (constant), so overall run time will be O(n^3)